src/components/ConnectionModal/utils.ts

```diff
@@ -3,13 +3,14 @@ import { ConnectionModalState } from './ConnectionModal.types'
 export function getConnectionMessage(connectionState: ConnectionModalState) {
   switch (connectionState) {
     case ConnectionModalState.ERROR: {
-      return 'You rejected the request in your <b>Wallet</b>. To continue, please try again.'
-    }
-    case ConnectionModalState.CONNECTING_WALLET: {
-      return 'To move forward, confirm the connect action in your <b>Wallet</b>.'
+      return 'You did not confirm this action in your digital wallet extension. To continue, please try again.'
     }
+    case ConnectionModalState.CONNECTING_WALLET:
     case ConnectionModalState.WAITING_FOR_SIGNATURE: {
-      return 'To move forward, confirm the signature action in your <b>Wallet</b>.'
+      return 'Confirm in your digital wallet extension to continue.'
```

Comment:
Signatures can be performed either from Magic or from an extension wallet, should we write `extension wallet` here?